### DISCUSSION TITLE
Using under windows - mouse to pan and zoom?

### DISCUSSION THREAD

**Discussion by nirgivon** [imaNNeo/fl_chart#1365] [General]
Hi,
I am looking to use fl_chart on a windows application.
my application analyzes log files from arduplane autopilot log files.

These log files contain many data points and produce large data sets.
Is there any kind of support for zooming / panning?
especially with a mouse

Also, can you give any feedback on using large data sets?

**Comment by imaNNeo** [comment]
You can follow this one: #71
I suggest to simplify your data, because we do not do any simplifications inside the chart
